Lib/test/test_zipimport.py

```diff
@@ -665,6 +668,18 @@ def testBytesPath(self):
         with self.assertRaises(TypeError):
             zipimport.zipimporter(memoryview(os.fsencode(filename)))
 
+    def testComment(self):
+        files = {TESTMOD + ".py": (NOW, test_src)}
+        self.doTest(".py", files, TESTMOD, comment=b"comment")
```

Comment:
Do comments *have* to be bytes?  If so, I think we should document that.  If not, let's test it.

Reply:
`doTest()` creates test zip files with `zipfile.ZipFile`, and the `comment` setter for `zipfile.ZipFile` requires bytes.